### PR TITLE
Shutdown timeout

### LINKED ISSUE #8755: Add `core.shutdown_timeout` as a way to override the default 5min timeout
When LXD shuts down for an update and there are running operations, it will wait for up to 5min for those to complete before moving on. This is implemented to be nice to `lxc exec` users but it being hardcoded to 5min can sometimes be too long or too short depending on environments.

Let's add a `core.shutdown_timeout` global config option and use that to override the default 5min timeout.

### PR DESCRIPTION
**Pull request by presztak**
Resolves #8755

### CODE REVIEW

**Comment by stgraber** [comment]
jenkins: test this please

File: lxd/operations.go [hunk 1]
```diff
@@ -54,7 +54,20 @@ var operationWebsocket = APIEndpoint{
...
+	err := s.Cluster.Transaction(func(tx *db.ClusterTx) error {
+		config, err := cluster.ConfigLoad(tx)
+		if err != nil {
+			return err
+		}
+		timeout = time.After(config.ShutdownTimeout())
+		return nil
+	})
+	if err != nil {
+		// If something goes really bad just set timeout to 5 minutes
```
Comment (tomponline) [inline lxd/operations.go:67]: @presztak please could you log an error here as well as setting the default timeout so we can see if something goes wrong. Thanks. e.g `logger.Error("Failed getting shutdown timeout", log.Ctx{"err": err})`

Comment (presztak) [reply lxd/operations.go:67]: @tomponline logging implemented.

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**